libkbfs/key_manager_test.go

```diff
@@ -1638,6 +1670,14 @@ func testKeyManagerRekeyBit(t *testing.T, ver MetadataVer) {
 		t.Fatalf("Device 2 couldn't read a: %+v", err)
 	}
 
+	// Cancel the original promptPaper timer for user2 now that the
+	// TLF is readable; otherwise the rekey later on will hang because
+	// it looks like prompt for paper was already set.  TODO: should
```

Comment:
Good point! Do you think it's better to just stop the timer, or unset the promptPaper flag in this case? If we just unset promptPaper, it would be a no-op when the timer is up if it's not needed, but still covers the case where a rekey is actually needed again. But I guess that's rare enough to ignore here?